### PR TITLE
Public docs update (e108) - variation and VEP

### PR DESCRIPTION
**Pull request by nakib103**
The changes can be seen in the sandbox - http://wp-np2-11.ebi.ac.uk:7070/

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 1]
```diff
@@ -115,6 +115,26 @@ <h2 id="new">What's new?</h2>
...
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
+  <ul>
+    <li>New plugin (supported on CLI, REST, and web): </li>
+      <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/mutfunc.pm" rel="external">mutfunc</a> - predicts destabilization of protein structure, interaction etc. by a variant (GRCh38 only)</li>
+      </ul>
+    <li>Plugin feature extension: </li>
+      <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/IntAct.pm" rel="external">IntAct</a> - 4 new species are now supported - rat, chicken (red jungle fowl), yeast, and arabidopsis</li>
```
Comment (diegomscoelho) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:125]:
```suggestion
        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/IntAct.pm" rel="external">IntAct</a> - 4 new species are now supported - rat, chicken (red jungle fowl), yeast and arabidopsis</li>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 2]
```diff
@@ -115,6 +115,26 @@ <h2 id="new">What's new?</h2>
 
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
+  <ul>
+    <li>New plugin (supported on CLI, REST, and web): </li>
+      <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/mutfunc.pm" rel="external">mutfunc</a> - predicts destabilization of protein structure, interaction etc. by a variant (GRCh38 only)</li>
```
Comment (diegomscoelho) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:121]:
```suggestion
        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/mutfunc.pm" rel="external">mutfunc</a> - predicts destabilization of protein structure, interaction and others features by a variant (GRCh38 only)</li>
```

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h1 style="margin-top:15px">Ensembl Variation - Pathogenicity predictions</h1>
 <p>
 Pathogenicity scores from <a href="#REVEL">REVEL</a>, <a href="#MutationAssessor"> MutationAssessor</a> 
 and <a href="#MetaLR">MetaLR</a> are also available for human missense variants.
-To improve compatibility with other resources, these scores are taken from dbNSFP (version 4.3a). 
+To improve compatibility with other resources, these scores are taken from dbNSFP (version 4.0a).
```
Comment (diegomscoelho) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html:46]: We downgraded this?

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 4]
```diff
@@ -97,7 +97,7 @@ <h2 id="CADD">CADD</h2>
       over 30 are predicted to be the 0.1% most deleterious possible substitutions in the human genome.<br />
       <b>We strongly recommend the actual score is used when assessing a variant and a cut-off appropriate to your requirements is chosen.</b>
     </p>
-    <p>CADD scores are taken from the <a rel="external" href="https://cadd.gs.washington.edu/download">CADD</a> project web site (version v1.6) for
+    <p>CADD scores are taken from the <a rel="external" href="https://cadd.gs.washington.edu/download">CADD</a> project web site (version v1.5) for
```
Comment (diegomscoelho) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html:100]: We also downgraded this one?

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html [hunk 5]
```diff
@@ -505,10 +499,10 @@ <h2 id="references">References</h2>
...
 
   <li>
-    <p>Liu X, Li C, Mou C, Dong Y, and Tu Y.<br />
-    <strong>dbNSFP v4: a comprehensive database of transcript-specific functional predictions and annotations for human nonsynonymous and splice-site SNVs.</strong><br />
-    <em>Genome Medicine 12</em> 103 (2020)<br />
-    <a href="https://doi.org/10.1186/s13073-020-00803-9" rel="external">doi:10.1186/s13073-020-00803-9</a></p>
+    <p>Liu, X., Wu, C., Li, C., Boerwinkle, E.<br />
+    <strong> dbNSFP v3.0: A One-Stop Database of Functional Predictions and Annotations for Human Non-synonymous and Splice Site SNVs.</strong><br />
+    <em> Human Mutation</em> 37(3), 235-241 (2016)<br />
+    <a href="http://doi.org/10.1002/humu.229326" rel="external">doi:10.1002/humu.229326</a></p>
```
Comment (diegomscoelho) [inline ensembl/htdocs/info/genome/variation/prediction/protein_function.html:505]: Doi is not working.

**Review by diegomscoelho — COMMENTED** [review]
Mostly OK. Gave a few suggestions. Let me know when everything is OK.

File: ensembl/htdocs/info/genome/variation/prediction/protein_function.html (hunk 5 above)
Comment (nakib103) [reply ensembl/htdocs/info/genome/variation/prediction/protein_function.html]: Thanks, it should work now.

**Review by diegomscoelho — APPROVED** [review]
LOTM!

**Pull request merged**